examples/mic_vad_streaming/mic_vad_streaming.py

```diff
@@ -1,41 +1,77 @@
 import time, logging
 from datetime import datetime
 import threading, collections, queue, os, os.path
-import wave
+import deepspeech
+import numpy as np
 import pyaudio
+import wave
 import webrtcvad
 from halo import Halo
-import deepspeech
-import numpy as np
+from scipy import signal
 
 logging.basicConfig(level=20)
 
+
 class Audio(object):
     """Streams raw audio from microphone. Data is received in a separate thread, and stored in a buffer, to be read from."""
 
     FORMAT = pyaudio.paInt16
-    RATE = 16000
+    # Network/VAD rate-space
+    RATE_PROCESS = 16000
     CHANNELS = 1
     BLOCKS_PER_SECOND = 50
-    BLOCK_SIZE = int(RATE / float(BLOCKS_PER_SECOND))
 
-    def __init__(self, callback=None):
+    def __init__(self, callback=None, device=None, input_rate=RATE_PROCESS):
         def proxy_callback(in_data, frame_count, time_info, status):
             callback(in_data)
             return (None, pyaudio.paContinue)
-        if callback is None: callback = lambda in_data: self.buffer_queue.put(in_data)
+        if callback is None: 
```

Comment:
nit: useless change

Reply:
Ah, my company convention has always been to avoid single liners like these, but I will revert if you prefer.